ChartsDemo/Swift/Demos/LineChart1ViewController.swift

```diff
@@ -126,7 +126,7 @@ class LineChart1ViewController: DemoBaseViewController {
         set1.valueFont = .systemFont(ofSize: 9)
         set1.formLineDashLengths = [5, 2.5]
         set1.formLineWidth = 1
-        set1.formLineWidth = 15
+        set1.formSize = 15
```

Comment:
why set formSize here but not just delete it?

Reply:
The objective-c version sets `formSize`

Reply:
oh I don't see it's swift controller.